src/deprecated.jl

```diff
@@ -9,7 +9,7 @@ const FINALIZE_ATEXIT = Ref(true)
 @deprecate Info_get(info::Info,key::AbstractString) info[Symbol(key)] false
 @deprecate Info_delete(info::Info,key::AbstractString) delete!(info, Symbol(key)) false
 @deprecate Info_free(info::Info) free(info) false
-
+@deprecate Cart_coords!(comm::Comm, maxdims::Integer) Cart_coords(comm, maxdims) false
```

Comment:
@simonbyrne : is this how it should be (coords allocated in the function and then returned)?

Reply:
Yes, that looks good, thanks.